.escheckrc

```diff
@@ -3,7 +3,8 @@
     "modules": "false",
     "files": "./dist/**/*.js",
     "not": [
-        "./dist/libraries/pdf.worker.js",
+        "./dist/libraries/pdf.worker.mjs",
+        "./dist/node_modules.pdfjs-dist.*",
```

Comment:
This file includes dynamic imports that are only used in `nodejs` environments, so it _should_ be safe to ignore.